Update the tracking budget page so the feature can be released. (#483)
actualbudget/docs: docs/getting-started/tracking-budget.md

# Tracking Budget

## What is the Tracking Budget

The **Tracking budget** option allows the user to shift from the default **Envelope budget** option to a more traditional style of budgeting.
The **Tracking budget** won't automatically rollover funds from month to month, and you don't have to account for all on budget funds continually.
Instead of working with what funds you have _now_, you can forecast expenses and income.
All transactions do still need to be categorized.

> We the Actual team suggest that you try to use the **Envelope Budget** if you haven't already.
> We believe it to be a more capable style of budgeting.
> If you still would prefer the more traditional **Tracking Budget** then you can switch to that if you would like.

Reviewer: Something read a bit strangely here, not sure if this is better though - what do you reckon?

```suggestion
We, the Actual team, suggest that you try to use the **Envelope Budget** if you haven't already as we believe it to be a more capable style of budgeting than the **Tracking Budget**. However, if you would prefer the more traditional style, then the **Tracking Budget** is for you.
```

:::note
If you utilize the **Rollover Overspending** feature on a category that is not over spent, the entire spent amount will be forwarded to the next month and the **Saved** amount will increase for the current month.
This will affect your spent totals as if the spending didn't happen.
The spending will only show up in the month that the rollover stops.
:::

<<<<<<< HEAD:docs/getting-started/tracking-budget.md
## Other Features
=======
## Working With the Budget

> >>>>>>> master:docs/experimental/tracking-budget.md

Reviewer 2: FYI looks like you have some merge conflict stuff that got left in
